src/daily-auths-report.tsx

```diff
@@ -279,29 +281,15 @@ function DailyAuthsReport(): VNode {
 
   return (
     <div ref={ref}>
-      <div className="usa-accordion usa-accordion--bordered margin-top-2 margin-bottom-2">
-        <h3 className="usa-accordion__heading">
-          <button
-            className="usa-accordion__button"
-            aria-controls="how-is-it-measured"
-            aria-expanded="false"
-            type="button"
-          >
-            How is this measured?
-          </button>
-        </h3>
-        <div className="usa-prose usa-accordion__content" id="how-is-it-measured" hidden>
-          <p>
-            <strong>Timing: </strong>
-            All data is collected, grouped, and displayed in the UTC timezone.
-          </p>
-          <p>
-            <strong>Counting: </strong>
-            This report displays the total number of authentications, so one user authenticating
-            twice will count twice. It does not de-duplicate users or provide unique auths.
-          </p>
-        </div>
-      </div>
+      <Accordion id="how-is-it-measured" title="How is this measured?">
+        <Markdown
```

Comment:
Is there a reason we want this Markdown component, vs. just rendering the markup?

Reply:
I felt lazy about writing HTML for prose, I don't feel that strongly about it, I can go back to HTML markup if you'd prefer